packages/cryptoplease/lib/app/components/status_screen.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:cryptoplease/gen/assets.gen.dart';
+import 'package:cryptoplease_ui/cryptoplease_ui.dart';
+import 'package:flutter/material.dart';
+
+class StatusScreen extends StatelessWidget {
```

Comment:
It also has the status itself. Let's pass `statusTitle` and `statusContent` and build the status widget inside.